pkg/kubelet/cm/dra/state/state_checkpoint.go

```diff
@@ -60,6 +60,9 @@ type ClaimInfoState struct {
 	// CdiDevices is a list of CDI devices returned by the
 	// GRPC API call NodePrepareResource
 	CdiDevices []string
+
+	// ClassName is a resource class of the claim
+	ClassName string
```

Comment:
I'd squash this commit in the previous one which was adding checkpoint support (can also apply to the former comment)

Reply:
Agreed -- this should be part of the checkointing PR